packages/frontpage/app/(app)/_components/share-button.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import { DropdownMenuItem } from "@/lib/components/ui/dropdown-menu";
+import { toast } from "@/lib/components/ui/use-toast";
+import { Share2Icon } from "@radix-ui/react-icons";
+
+export function ShareDropdownButton({ path }: { path: string }) {
+  return (
+    <DropdownMenuItem
+      onSelect={() => {
+        void navigator.clipboard
+          .writeText(new URL(path, location.href).toString())
```

Comment:
I think this is ok for now might I might look at trying to trigger the share menu https://developer.mozilla.org/en-US/docs/Web/API/Web_Share_API